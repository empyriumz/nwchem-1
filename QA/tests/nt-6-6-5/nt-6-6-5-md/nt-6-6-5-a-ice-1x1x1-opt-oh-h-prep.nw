echo 
start md_dat

prepare
  system nt-ice_md
  new_rst
  source ja4074529_si_004.pdb
end

task prepare

