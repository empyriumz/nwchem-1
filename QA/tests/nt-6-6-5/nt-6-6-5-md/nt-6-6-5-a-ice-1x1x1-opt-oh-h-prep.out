 argument  1 = nt-6-6-5-a-ice-1x1x1-opt-oh-h-prep.nw



============================== echo of input deck ==============================
echo
start md_dat

prepare
  system nt-ice_md
  new_rst
  source ja4074529_si_004.pdb
end

task prepare

================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.6
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2015
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = hpc1.csc.bnl.gov
    program         = ../../../../bin/LINUX64/nwchem
    date            = Fri May  5 21:47:36 2017

    compiled        = Tue_Apr_25_15:08:50_2017
    source          = /home/hvandam/tau/nwchem-mpi-mt
    nwchem branch   = Development
    nwchem revision = 29056
    ga revision     = 10752
    input           = nt-6-6-5-a-ice-1x1x1-opt-oh-h-prep.nw
    prefix          = md_dat.
    data base       = ./md_dat.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =  134217728 doubles =   1024.0 Mbytes
    stack    =  134217725 doubles =   1024.0 Mbytes
    global   =  268435456 doubles =   2048.0 Mbytes (distinct from heap & stack)
    total    =  536870909 doubles =   4096.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                                  Prepare Module
                                  --------------

 Force field                           amber


 Directories used for fragment and segment files

                                       /home/hvandam/tau/nwchem-dev/QA/../src/data/amber_s/
                                       /home/hvandam/tau/nwchem-dev/QA/../src/data/amber_x/
                                       ./

 Parameter files used to resolve force field parameters

                                       /home/hvandam/tau/nwchem-dev/QA/../src/data/amber_s/amber.par
                                       /home/hvandam/tau/nwchem-dev/QA/../src/data/amber_x/amber.par
                                       ./amber.par
 Using existing topology               nt-ice.top

 Deleted existing restart              nt-ice_md.rst

 Topology                              nt-ice.top

 No command file found: Default restart directives

 Z-matrix definitions done

 Geometric rules definitions done

 Grid specifications for non-bonded atoms done

 Crystal waters done

 Solute centered in x-dimension
 Solute centered in y-dimension
 Solute centered in z-dimension

 Boxsize determined to                     3.552419    3.780913    6.202034


 Created restart                       nt-ice_md.rst


 Task  times  cpu:        0.1s     wall:        0.4s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:    0        0        0        0        0        0        0        0     
number of processes/call 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
bytes total:             0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 0 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	         7	        31
	current total bytes		         0	         0
	maximum total bytes		      1136	   3795912
	maximum total K-bytes		         2	      3796
	maximum total M-bytes		         1	         4


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
       S. A. Fischer, S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen,
    V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler, V. Konjkov, T. Risthaus,
       M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols,
      R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, M. Williamson,
      R. J. Harrison, J. R. Rehr, M. Dupuis, D. Silverstein, D. M. A. Smith,
            J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        0.2s     wall:        4.7s
